### PR TITLE
Add ceph_keyring_permissions var to set permissions for keyring files

### PR DESCRIPTION
**Pull request by amarao**
Add `keyring_permissions` variable to control permissions for keyring
files in `/etc/ceph`. Default value is the same as it was (0600),
but this variable allows user to override it (f.e. set it to 0640).

Rationale: We are configuring a monitoring for Ceph. It runs under
a separate user (telegraf), but this user is included into the ceph
group. To have access to the admin keyring it need to have
/etc/ceph/client.admin.keyring to be at least rw-r----- (0640).
I added a variable into ceph-defaults with the same value as current
value in all related tasks.  This will allow to override it through variable
in the inventory/group_vars/host_vars.

### CODE REVIEW

File: roles/ceph-defaults/defaults/main.yml [hunk 1]
```diff
@@ -288,6 +288,9 @@ generate_fsid: true
 
 ceph_conf_key_directory: /etc/ceph
 
+# Permissions for keyring files in /etc/ceph
+keyring_permissions: '0640'
```
Comment (leseb) [inline roles/ceph-defaults/defaults/main.yml]: Do you mind prefixing with `ceph_`? Thanks.

File: group_vars/rhcs.yml.sample [hunk 2]
```diff
@@ -296,6 +296,9 @@ ceph_repository: rhcs
 
 #ceph_conf_key_directory: /etc/ceph
 
+# Permissions for keyring files in /etc/ceph
+#keyring_permissions: '0640'
```
Comment (andymcc) [inline group_vars/rhcs.yml.sample]: Default should be 0600 to ensure no change from current setup.

**Comment by andymcc** [comment]
Thanks for changing it up - looks better this way.
2 minor changes and then lgtm!

File: group_vars/rhcs.yml.sample (hunk 2 above)
Comment (amarao) [reply group_vars/rhcs.yml.sample]: Oh! Sorry, slipped from my tests. I'll set it to `0600`.

File: roles/ceph-defaults/defaults/main.yml (hunk 1 above)
Comment (amarao) [reply roles/ceph-defaults/defaults/main.yml]: done.

**Comment by leseb** [comment]
@andymcc lgty?

**Comment by amarao** [comment]
Thank you for advice! I've learned a lot.

**Pull request merged**